etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/sftp/SFTP.java

```diff
@@ -94,7 +93,7 @@ private Session connectedKeyBasedSFTPSession(String user, String host, int port,
     }
 
     private void transferFile(Session session, String remoteFile, String localFile) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "transferFile" is 20 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)